sdk/storage/storage-file-share/src/generatedModels.ts

```diff
@@ -52,5 +52,6 @@ export {
   SignedIdentifier as SignedIdentifierModel,
   SourceModifiedAccessConditions,
   FileForceCloseHandlesHeaders,
-  DirectoryForceCloseHandlesHeaders
+  DirectoryForceCloseHandlesHeaders,
+  CopyFileSmbInfo,
```

Comment:
Can we rename this to `FileSmbProperties` to be consistent with other lagnuages?

Reply:
They are also using `CopyFileSmbInfo`. `FileSmbProperties ` is their way of wrapping. We don't have it for now.
```
        final CopyFileSmbInfo copyFileSmbInfo = new CopyFileSmbInfo()
            .setFilePermissionCopyMode(filePermissionCopyMode)
            .setFileAttributes(fileAttributes)
            .setFileCreationTime(fileCreationTime)
            .setFileLastWriteTime(fileLastWriteTime)
            .setIgnoreReadOnly(ignoreReadOnly)
            .setSetArchiveAttribute(setArchiveAttribute);
```